_notebooks/2020-12-28-CoreferenceResolution.ipynb

```diff
@@ -0,0 +1,230 @@
+{
```

Comment:
<p>Please mention that the first import of neuralcoref is going lead to a large download in the comments?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/projectnoor/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>